L.Control.Geonames.js

```diff
@@ -27,7 +27,7 @@ L.Control.Geonames = L.Control.extend({
     },
     onAdd: function() {
         this._container = L.DomUtil.create('div', 'leaflet-geonames-search leaflet-bar');
-        this._container.title = 'Search by location name';
+        //this._container.title = 'Search by location name'; //too redundant
```

Comment:
set this if `this.options.alwaysOpen` is `false`, but make it attached to the button (`this._link` on next line) instead of the container.